app/components/UI/DrawerView/index.js

```diff
@@ -474,16 +470,19 @@ class DrawerView extends PureComponent {
   previousBalance = null;
   processedNewBalance = false;
   animatingNetworksModal = false;
+  selectedCheckSummedAddress = toChecksumHexAddress(
```

Comment:
nit: I think checksum is one word

```suggestion
  selectedChecksummedAddress = toChecksumHexAddress(
```